.gitignore

```diff
@@ -12,4 +12,4 @@ node_modules/
 !.yarn/versions
 
 # macos files
-.DS_Store
+.DS_Store
```

Comment:
Is there a reason you are removing the empty line at the end of some files?

![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/2486142/74bc885f-0f93-4e53-b20a-9321e465d5cb)

There are a couple more (*.mjs extension)

Reply:
Actually I think this all files had conflicts, and when I resolved and saved my text editor autoformatted and removed the empty line 

Happy to revert these empty line changes 🙌 But after these changes none of the *.mjs files has an extra empty line at the end 

We should probably soon setup prettier for CLI I think

Reply:
We always have the empty line at the end since it's a good practice/convention...but we can leave it like this until we have prettier!

Thanks

Reply:
> We always have the empty line at the end since it's a good practice/convention...but we can leave it like this until we have prettier!

Woohhh, I see !! Just read about this, it also helps in getting clean diffs when adding content at end and also some other advantages !! Tysm !!